ui/src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -299,7 +299,7 @@ export const createOrUpdateTelegrafConfigAsync = (authToken: string) => async (
     type: TelegrafPluginOutputInfluxDBV2.TypeEnum.Output,
     config: {
       urls: ['http://127.0.0.1:9999'],
-      token: authToken,
+      token: '$INFLUX_TOKEN',
```

Comment:
So we won't store the exact token here anymore?  Does that mean the auth token will need to be fetched separately for the `TelegrafInstructionsOverlay` in the other PR I've got open where it was using `plugin.config.token`?

Reply:
https://github.com/influxdata/influxdb/pull/11677

Reply:
Oh yes i do believe that will affect the Instructions Overlay in that way. Since we are not going to store that anymore, fetching it will be the only to get that value.